examples/common/multi-container/values.yaml

```diff
@@ -64,3 +66,8 @@ common:
     enabled: true
     data:
       APP1CONF: yes
+
+  secrets:
+    auth-credentials: # k8s secret name "multi-container-auth-credentials"
```

Comment:
I'm not sure I like this comment. It may lead to our devs adding a `fromSecretRef` environment key, and they don't need to.

The k8s name should not really matter, what's important is that the secrets will be mounted to their ENV.

Reply:
I agree. This was added to help the developers understand the name of the secrets being created. But I think it can create more misunderstanding to have the comment there.